internal/services/kaas/kaas_resource_test.go

```diff
@@ -70,13 +69,13 @@ func TestKaasResource_Schema(t *testing.T) {
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
+			tc.IsUnitTest = true
```

Comment:
Just a nitpick, for parity

```suggestion
    for name, tc := range testCases {
    	tc.IsUnitTest = true
        t.Run(name, func(t *testing.T) {
```